lib/tmuxinator/cli.rb

```diff
@@ -210,6 +210,17 @@ def render_project(project)
         Kernel.exec(project.render)
       end
 
+      def version_warning?(quiet)
+        !Tmuxinator::TmuxVersion.supported? && !quiet
+      end
+
+      def show_version_warning
+        say Tmuxinator::TmuxVersion::UNSUPPORTED_VERSION_MSG, :red
+        say
```

Comment:
Is this second `say` necessary?

Reply:
This is just copy pasted from the deprecation warnings to keep the blank line between the message and the confirmation:
https://github.com/tmuxinator/tmuxinator/blob/8dab40337d85ee27ec27bf34cbdfb3290a0d5c72/lib/tmuxinator/cli.rb#L205
I guess it's easier to read this way?

But let me pull out the three duplicate lines, so that this would only need to change in one place.